stdlib/jdbc/src/main/java/org/ballerinax/jdbc/methods/ExternFunctions.java

```diff
@@ -72,9 +74,9 @@ public static void createClient(ObjectValue client, MapValue<String, Object> con
         String dbType = url.split(":")[1].toUpperCase(Locale.getDefault());
 
         SQLDatasource.SQLDatasourceParamsBuilder builder = new SQLDatasource.SQLDatasourceParamsBuilder(dbType);
-        SQLDatasource.SQLDatasourceParams sqlDatasourceParams = builder.withPoolOptions(poolOptionsWrapper)
-                .withJdbcUrl(url).withUsername(username).withPassword(password).withDbOptionsMap(dbOptions)
-                .withIsGlobalDatasource(userProvidedPoolOptionsNotPresent).build();
+        SQLDatasource.SQLDatasourceParams sqlDatasourceParams = builder.withPoolOptions(poolOptionsWrapper).
+                withJdbcUrl(url).withUsername(username).withPassword(password).withDbOptionsMap(dbOptions).
+                withIsGlobalDatasource(userProvidedPoolOptionsNotPresent).build();
```

Comment:
What is the reason for appending `.` at the end instead of the beginning of the line?